development/build-image.sh

```diff
@@ -55,7 +55,7 @@ echo "Final image name: $USERNAME/$IMAGE:$TAG"
 
 if [[ $TAG = "develop-$CONFIG_NAME" ]]; then
     git checkout -- docker-compose.yml
-    sed -i "s|usdotfhwastol|$USERNAME|g; s|:[0-9]*\.[0-9]*\.[0-9]*|:develop|g; s|:CARMASystem_[0-9]*\.[0-9]*\.[0-9]*|:develop|g;" \
+    sed -i "s|usdotfhwastol/|$USERNAME/|g; s|:[0-9]*\.[0-9]*\.[0-9]*|:develop|g; s|:CARMASystem_[0-9]*\.[0-9]*\.[0-9]*|:develop|g;" \
         docker-compose.yml
```

Comment:
should this line be removed as well?

Reply:
I don't think it needs to be. It's fixed to not turn usdotfhwastoldev into usdotfhwastoldevdev, and this way if for whatever reason it's pointing to our main repo, build-image.sh -d will handle that